OrbitGl/TimeGraph.cpp

```diff
@@ -38,7 +38,11 @@
 TimeGraph* GCurrentTimeGraph = nullptr;
 
 //-----------------------------------------------------------------------------
-TimeGraph::TimeGraph() { m_LastThreadReorder.Start(); }
+TimeGraph::TimeGraph() {
+  m_LastThreadReorder.Start();
+  scheduler_track_ = std::make_shared<SchedulerTrack>(this);
+  process_track_ = GetOrCreateThreadTrack(0);
```

Comment:
Add a comment to indicate what the "0" here is.

Reply:
Done.